src/lib.rs

```diff
@@ -223,57 +223,55 @@ pub struct BigBrainPlugin;
 
 impl Plugin for BigBrainPlugin {
     fn build(&self, app: &mut App) {
-        use CoreStage::*;
+        app.configure_sets((
+            BigBrainSet::Scorers.in_base_set(CoreSet::First),
+            BigBrainSet::Thinkers
+                .in_base_set(CoreSet::First)
+                .after(BigBrainSet::Scorers),
+            BigBrainSet::Actions.in_base_set(CoreSet::PreUpdate),
```

Comment:
I've been kinda wondering if `PreUpdate` is really the appropriate stage for this, to begin with...

Reply:
It's definitely a safe option, but I can't say if it's appropriate or not. Ideally users would just be able to specify which set it would be in, but I'm not sure if that's possible by default with sets.